pkg/scheduler/core/equivalence_cache.go

```diff
@@ -122,17 +123,14 @@ func (ec *EquivalenceCache) PredicateWithECache(
 		if cachePredicate, exist := algorithmCache.predicatesCache.Get(predicateKey); exist {
 			predicateMap := cachePredicate.(PredicateMap)
 			// TODO(resouer) Is it possible a race that cache failed to update immediately?
```

Comment:
Mind delete this comment? It should have been fixed by previous patch.

Reply:
Done